qa/workunits/rgw/s3_utilities.pm

```diff
@@ -134,15 +134,31 @@ sub purge_data
     return 0;
 }
 
+# Read PRETTY_NAME from /etc/os-release
+sub os_pretty_name
+{
+        open(FH, '<', '/etc/os-release') or die $!;
```

Comment:
might want to use 4 space for indent to be more consistent with the rest of this source file.